code/AdobeEdge.brs

```diff
@@ -585,6 +589,7 @@ function _adb_StateManager() as object
                 m._edge_configId = configId
             end if
 
+            ' example domain: org.data.edge.adobedc.net
```

Comment:
It should be `org.data.adobedc.net` per https://developer.adobe.com/client-sdks/documentation/edge-network/#domain-configuration.